src/main/java/mekanism/common/content/gear/mekasuit/ModuleMekaSuit.java

```diff
@@ -398,4 +404,18 @@ public void addHUDElements(List<HUDElement> list) {
             }
         }
     }
+    
+    public static class ModuleGeigerUnit extends ModuleMekaSuit {
+        private static final ResourceLocation icon = MekanismUtils.getResource(ResourceType.GUI_HUD, "geiger_counter.png");
+        @Override
+        public void addHUDElements(List<HUDElement> list) {
+            if (!isEnabled()) {
+                return;
+            }
+                double magnitude = MekanismAPI.getRadiationManager().getRadiationLevel(Minecraft.getInstance().player);
+                HUDElement e = HUDElement.of(icon, UnitDisplayUtils.getDisplayShort(magnitude, RadiationUnit.SV, 3));
```

Comment:
Overall looks good and seems to work quite well so thank you for the PR. One last minor thing though is that this `, 3` should be changed to a `, 2` to match how we show the dosimeter's level on the HUD as the extra decimal point of precision isn't needed on the HUD.